PostProcessing/Editor/PostProcessLayerEditor.cs

```diff
@@ -162,6 +162,9 @@ void DoAntialiasing()
                     if (RuntimeUtilities.isSinglePassStereoSelected)
                         EditorGUILayout.HelpBox("TAA requires Unity 2017.3+ for Single-pass stereo rendering support.", MessageType.Warning);
                     #endif
+                    var camera = m_Target.GetComponent<Camera>();
+                    if (camera.allowDynamicResolution)
```

Comment:
Should this be #ifdef'd with UNITY_2019_1_OR_NEWER?

Reply:
I don't believe so, allowDynamicResolution wasn't new in 2019.1 and exists in 2017.4